File: u3u2/Assets/Scripts/app/net/c2s/CGFirstMarry.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 申请结婚,弹框
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGFirstMarry :BaseMessage
{
	
	
	public CGFirstMarry ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_FIRST_MARRY;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: IdeaServer/zlj2019/game_server/src/com/imop/lj/gameserver/marry/msg/CGFirstMarry.java
package com.imop.lj.gameserver.marry.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.marry.handler.MarryHandlerFactory;

/**
 * 申请结婚,弹框
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGFirstMarry extends CGMessage{
	
	
	public CGFirstMarry (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_FIRST_MARRY;
	}
	
	@Override
	public String getTypeName() {
		return "CG_FIRST_MARRY";
	}


	@Override
	public void execute() {
		MarryHandlerFactory.getHandler().handleFirstMarry(this.getSession().getPlayer(), this);
	}
}